app/views/courses/confirmation.html.erb

```diff
@@ -8,16 +8,194 @@
     <%= form_with model: course,
                   url: new_provider_recruitment_cycle_course_path(@provider.provider_code, @provider.recruitment_cycle_year),
                   method: :post do |f| %>
-
       <hr class="govuk-section-break govuk-section-break--l">
+      <div class="govuk-grid-row govuk-!-margin-bottom-9">
```

Comment:
Any way of sharing the markup from the basic details view?

Reply:
Potentially - but I held off prematurely generalising it for now. Gut feeling is that by prematurely trying to share it, those change links are going to be more difficult to implement. 

I'd rather see what ends up being shared, and what doesn't, and generalise from there

Reply:
Good with me